app/models/user.rb

```diff
@@ -1,7 +1,7 @@
 class User < ApplicationRecord
-  has_many :topics, through: :user_topics
-  # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable and :omniauthable
+  has_many :user_lessons
+
```

Comment:
`has_many :lessons`
and perhaps we could add:
`has_many :topics, through: :lessons`
also might be handy to add but not sure on correctness:
`has_many :categories, through: :topics, source: :lessons`